source/Engine.cpp

```diff
@@ -1695,9 +1756,19 @@ void Engine::CalculateStep()
 
 
 
+// Move a ship. Can be called concurrently.
```

Comment:
```suggestion
// Wrapper for MoveShip() that can be called in a concurrent context.
```
or use the full actual comment:
```
// Move a ship. Also determine if the ship should generate hyperspace sounds or
// boarding events, fire weapons, and launch fighters.
```